netlink-request/src/lib.rs

```diff
@@ -26,6 +23,19 @@ mod linux {
         };
     }
 
+    pub fn get_max_netlink_buffer_length() -> usize {
+        // https://www.kernel.org/doc/html/v6.2/userspace-api/netlink/intro.html#buffer-sizing
+        // "Netlink expects that the user buffer will be at least 8kB or a page
+        // size of the CPU architecture, whichever is bigger.""
+        const MIN_NELINK_BUFFER_LENGTH: usize = 8 * 1024;
+        let page_size = sysconf(SysconfVar::PAGE_SIZE).unwrap().unwrap() as usize;
```

Comment:
Do you think we should use a default here instead of double unwrapping?

Reply:
This shouldn't ever fail unless something has gone horribly wrong, so it's probably safer to just unwrap; I added a comment clarifying this.